Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -447,6 +447,10 @@ void NativeReanimatedModule::unregisterSensor(
   animatedSensorModule.unregisterSensor(sensorId);
 }
 
+void NativeReanimatedModule::cleanupSensors() {
+  animatedSensorModule.unregisterAllSensors();
+}
```

Comment:
Can we rename `cleanupSensors` to `unregisterAllSensors` for the sake of consistency?

Reply:
I think `cleanup` is a better naming here, since the caller doesn't really care what `NativeReanimatedModule` actually needs to do to cleanup.
I'd be more willing to rename the method on `AnimatedSensorModule` to `cleanup` as well, though again, there `unregister` is more consistent with the already present `unregisterSensor`

Reply:
Ok, let it be.